apps/demos/Demos/FileUploader/CustomDropzone/Angular/app/app.component.ts

```diff
@@ -24,6 +24,8 @@ export class AppComponent {
 
   progressValue = 0;
 
+  allowedFileExtensions: string[] = ['.jpg', '.jpeg', '.gif', '.png'];
```

Comment:
if we create an instance property here, let's also use it in markup `app.component.html`